cypress/integration/courses/02-courses.js

```diff
@@ -126,15 +126,15 @@ describe('ui-courses: course creation, editing and deletion', () => {
     it('adds instructors', () => {
       // No plugin available in unit test, so we have to enter by hand
       cy.contains('0 instructors')
-      cy.contains('Edit instructor').should('not.exist')
+      cy.get('[icon=edit]').should('not.exist')
```

Comment:
Out of interest, why this change? It's defined that tests run under the `en` locale, isn't it?

Reply:
Because I forgot to change the tests when I changed the `Edit instructor` buttons a while back to just be the edit `IconButton`s 🤦